src/main/java/uk/gov/ch/transformers/officer/bankrupt/BankruptOfficersTransformer.java

```diff
@@ -77,7 +77,12 @@ public ScottishBankruptOfficerSearchResult convertToSearchResult(ScottishBankrup
         details.setAddressLine3(scottishBankruptOfficerDetailsDataModel.getAddressLine3());
         details.setCounty(scottishBankruptOfficerDetailsDataModel.getAddressCounty());
         details.setTown(scottishBankruptOfficerDetailsDataModel.getAddressTown());
-        details.setDebtorDischargeDate(scottishBankruptOfficerDetailsDataModel.getDebtorDischargeDate());
+
+        LocalDate debtorDischargeDate = scottishBankruptOfficerDetailsDataModel.getDebtorDischargeDate();
+
+        if (debtorDischargeDate != null && !END_OF_TIME.isEqual(debtorDischargeDate)) {
```

Comment:
Do we need to check debtorDischargeDate is null
if we invoke method !END_OF_TIME.isEqual() or !END_OF_TIME.equals()